docs/cookbooks/BasicDataExtraction.md

```
@@ -48,25 +48,25 @@ class PersonalData(DataModel):
 
-Since we are going to generate a structured output we will use the `generate_model` function which automatically decodes the result of LLM call into a python object. We have to prepare the execution context, provide an instruction, source document and specify the data model and we can expect extracted data to be available. Keep in mind that depending on the task it might be beneficial to prepare a specific instruction and/or specify more details about the model itself.
+Since we are going to generate a structured output we will use the `ModelGeneration.generate` helper which automatically decodes the result of a model call into a python object. We have to prepare the execution context, provide instructions, the source document and specify the data model, and we can expect extracted data to be available. Keep in mind that depending on the task it might be beneficial to prepare a specific instruction and/or specify more details about the model itself.
 
 
 ```python
 from draive import ctx, ModelGeneration
-from draive.openai import OpenAIChatConfig, OpenAI
+from draive.openai import OpenAIResponsesConfig, OpenAI
 
 async with ctx.scope(
     "data_extraction",
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)